kloppy/_providers/wyscout.py

```diff
@@ -47,3 +64,19 @@ def load_open_data(
         coordinates=coordinates,
         event_factory=event_factory,
     )
+
+
+def identify_data_model_version(event_data):
+    with open(event_data) as json_file:
+        events = json.load(json_file)
+    events = events["events"]
+    if "eventName" in events[0]:
```

Comment:
I would store `events[0]` in a variable named `first_event` or something along those lines to improve the readability of the code.